keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+      break;
+    case NUMPAD:
+      if(record->event.pressed){
+	layer_on(_NUMPAD);
+	PORTB &= ~(1<<0);
+      } else {
+	layer_off(_NUMPAD);
+	PORTB |= (1<<0);
+      }
+      return false;
+      break;
+    case AE:           
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT("'"));
+      }    
+      break;
+    case EE:            
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT(";"));
+      }    
+      break;
+    case IE:            
+      if(record->event.pressed) {
+	SEND_STRING(SS_RALT("z"));
+      }    
+      break;
+    case VAI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_val_noeeprom();
+      }    
+      break;
+    case VAD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_val_noeeprom();
+      }    
+      break;
+    case SAI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_sat_noeeprom();
+      }    
+      break;
+    case SAD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_sat_noeeprom();
+      }    
+      break;
+    case HUI_NEE:            
+      if(record->event.pressed) {
+        rgblight_increase_hue_noeeprom();
+      }    
+      break;
+    case HUD_NEE:            
+      if(record->event.pressed) {
+        rgblight_decrease_hue_noeeprom();
+      }    
+      break;
+    case TOG_NEE:            
+      if(record->event.pressed) {
+	rgblight_toggle_noeeprom();
+      }    
+      break;
+    case RGB_ANM:            
+      if(record->event.pressed) {
+        rgblight_step_noeeprom();
+	rgb_mode = rgblight_get_mode();
+      }    
+      break;
+    case RGB_PRE:            
+      if(record->event.pressed) {
+        rgblight_step_reverse_noeeprom();
+	rgb_mode = rgblight_get_mode();
+      }    
+      break;
+    case SLIGHTLY:
+      if(record->event.pressed) {
+	SEND_STRING(":slightly_smiling_face:");
+      }    
+      break;
+    case SMILE:
+      if(record->event.pressed) {
+	SEND_STRING(":smile:");
+      }    
+      break;
+    case JOY:
+      if(record->event.pressed) {
+	SEND_STRING(":joy:");
+      }    
+      break;
+    case RELAXED:
+      if(record->event.pressed) {
+	SEND_STRING(":relaxed:");
+      }    
+      break;
+    case HEART:
+      if(record->event.pressed) {
+	SEND_STRING(":heart:");
+      }    
+      break;
+    case SAD:
+      if(record->event.pressed) {
+	SEND_STRING(":white_frowning_face:");
+      }    
+      break;
+    case CRY:
+      if(record->event.pressed) {
+	SEND_STRING(":cry:");
+      }    
+      break;
+    case NETRURAL:
+      if(record->event.pressed) {
+	SEND_STRING(":neutral_face:");
+      }    
+      break;
+    case SCREAM:
+      if(record->event.pressed) {
+	SEND_STRING(":scream:");
+      }    
+      break;
+    case THUMBSUP:
+      if(record->event.pressed) {
+	SEND_STRING(":+1:");
+      }    
+      break;
+  }
+  return true;
+}
+
+uint32_t layer_state_set_user(uint32_t state) {
```

Comment:
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
```

Though, really, you may want: 
```suggestion
layer_state_t layer_state_set_user(layer_state_t state) {
    state = update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
    writePin(B0, !(state & (1UL << (_NUMPAD))));     
```

The first additional line handles the tri layer stuff, without needing a keycode for it. However, it means that you can only get to the adjust layer via this means. However, it also means that you can use LT, and other layer functions that you normally couldn't. 

The second line sets the LED pin to high or low, depending on the `_NUMPAD` layer's state.